core/src/test/java/ai/timefold/solver/core/impl/testdata/domain/declarative/task_assignment/TestdataTAConstraintProvider.java

```diff
@@ -0,0 +1,27 @@
+package ai.timefold.solver.core.impl.testdata.domain.declarative.task_assignment;
+
+import java.time.Duration;
+
+import ai.timefold.solver.core.api.score.buildin.hardsoft.HardSoftScore;
+import ai.timefold.solver.core.api.score.stream.Constraint;
+import ai.timefold.solver.core.api.score.stream.ConstraintFactory;
+import ai.timefold.solver.core.api.score.stream.ConstraintProvider;
+
+import org.jspecify.annotations.NonNull;
+
+public class TestdataTAConstraintProvider implements ConstraintProvider {
```

Comment:
Just as with "FSR", "TA" is also meaningless here. The solver isn't tailored to any particular problem, and these domain classes should therefore carry generic names. We already have dozens of them, and we were able to follow this policy; I don't see the need to divert now.